src/viewer/meshLoader/meshLoader.ts

```diff
@@ -14,6 +14,7 @@ interface EntityMeshConfig {
 	scaleDamper?: number;
 	loadScale?: number;
 	color?: string;
+	engineOffsets: object | null;
```

Comment:
Don't use `object`. Should be an optional type so we don't have to put an empty array on every config object. The type should probably be `engineOffsets?: Vector[]`